app/assets/stylesheets/views/_brexit-landing-page.scss

```diff
@@ -78,7 +85,7 @@ $red: #E61E32;
 }
 
 .landing-page__section-content {
-  padding-left: 0;
+  padding-left: govuk-spacing(3);
```

Comment:
I don't think this is needed - this class is applied to a govuk column, which already has a padding left of 15px.